src/compat/libc/string/ffs.c

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file
+ * @brief Implementation of ffs() function.
+ *
+ * @date 20.03.15
+ * @author Byeonggon Lee
+ */
+
+int ffs(int i) {
```

Comment:
Are you sure this implementation is required? You can search for a similar thing in embox.

Reply:
I mean you can try to use existing bit functions.

Reply:
Please do not squash the commits next time :) You can squash them right before merge.

Now I see that `bit_ffs` uses `unsigned long` as an argument. But `ffs` uses int.

Reply:
So probably we cannot use `bit_ffs` here.

Reply:
So we should check something like `ffs(-1)` in the tests too.

Reply:
> Please do not squash the commits next time :) You can squash them right before merge
Okay I got it.